src/ibm/hypervisor-network-mgr-src/hyp_ethernet_interface.cpp

```diff
@@ -1003,7 +1003,7 @@ HypEthInterface::DHCPConf HypEthInterface::dhcpEnabled(DHCPConf value)
         bool v4Enabled = false;
         bool v6Enabled = false;
         bool slaacEnabled = false;
-        HypEthernetIntf::DHCPConf newValue;
+        HypEthernetIntf::DHCPConf newValue = HypEthernetIntf::DHCPConf::v4;
```

Comment:
Was this needed for the format work?

Reply:
seems its needed https://github.com/ibm-openbmc/phosphor-networkd/pull/116/commits/a3d268447e3dbfa9ffa4f64344843a17b9775dc5

Reply:
Sure. Thanks Andrew. I think this should be OK. 
But we will keep an eye on the same and will need @prkatti1 to run the VMI network test regression bucket after this.